frontend/src/components/NotifyModal.js

```diff
@@ -0,0 +1,141 @@
+import React, {useState, useRef, useEffect, useCallback} from 'react';
+import Modal from 'react-modal';
+import styled from 'styled-components'
+import {MdClose} from 'react-icons/md';
+//import {useSpring, animated} from 'react-spring;'
+
+function NotifyModal() {
+    const [modalIsOpen, setModalIsOpen] = useState(false)
+    
+    //create open modal function
+    const openModal = () => {
+        setModalIsOpen(prev => !prev);
+    };
```

Comment:
It looks like we're not using this, and the functionality is pretty simple, so I think we can take this out